### PR TITLE
Artifacts: Update of the Essentils document + Introduction of Retention Policies

### PR DESCRIPTION
**Pull request by shiroyasha**
In this PR, I am:

- Touching up the content of the artifacts. The new essentials page is now focused on "why you need artifacts, and what can you do with them".
- Adding screenshots to the artifacts page. App screenshots boost the "understandability" of the content.
- Introducing retention policies for controlling the lifetime of artifacts
- Introducing retention policies reference

---

Artifact retention policies are a new concept in Semaphore 2.0. They are not yet released as a feature. A presentation of this feature is due to the internal team before the feature is released.

For some of you in the review list, retention policies are a new concept. Your feedback is the most useful, as you have the unique opportunity to judge if the concept is understandable just based on the docs.

### CODE REVIEW

File: docs/essentials/artifacts.md [hunk 1]
```diff
@@ -1,115 +1,182 @@
...
+<img style="box-shadow: 0px 0px 5px #ccc" src="/essentials/img/artifacts/simple-retention-policy.png" alt="Simple retention policies">
+
+### Applying different retention policies to different folders
+
+Some artifacts have higher values and you might want to extend their lifetime. 
+
+For example, test screenshots saved on the job level typically lose their value 
+after a couple days, while the test logs can offer a high value for several months.
+
+We can set up a retention policy that matches that matches our previous needs. One
```
Comment (mateuscap) [inline docs/essentials/artifacts.md:163]:
```suggestion
We can set up a retention policy that matches our previous needs. One
```

File: docs/essentials/artifacts.md [hunk 2]
```diff
@@ -1,115 +1,182 @@
...
+week for screenshots, and 3 months for test logs. We will also set one months for
```
Comment (mateuscap) [inline docs/essentials/artifacts.md:164]:
```suggestion
week for screenshots, and 3 months for test logs. We will also set one month for
```

**Review by mateuscap — COMMENTED** [review]
_Your feedback is the most useful, as you have the unique opportunity to judge if concept is understandable just based on the docs._

The concept is understandable to me

**Review by csidyel — CHANGES_REQUESTED** [review]
I'll ask Christopher to review this page after the merge.

**Pull request merged**